projects/plugins/jetpack/class.json-api.php

```diff
@@ -1021,26 +1021,37 @@ public function wp_count_comments( $post_id ) {
 		);
 
 		/**
-		 * Exclude certain comment types from comment counts in the REST API.
-		 *
-		 * @since 6.9.0
-		 * @module json-api
-		 *
-		 * @param array Array of comment types to exclude (default: 'order_note', 'webhook_delivery', 'review', 'action_log')
-		 */
-		$exclude = apply_filters(
-			'jetpack_api_exclude_comment_types_count',
-			array( 'order_note', 'webhook_delivery', 'review', 'action_log' )
+		* Exclude certain comment types from comment counts in the REST API.
+		*
+		* @since 6.9.0
+		* @deprecated $$next-version$$
+		* @module json-api
+		*
+		* @param array Array of comment types to exclude (default: 'order_note', 'webhook_delivery', 'review', 'action_log')
+		*/
+		$exclude = apply_filters_deprecated( 'jetpack_api_exclude_comment_types_count', array( 'order_note', 'webhook_delivery', 'review', 'action_log' ), 'jetpack-11.0', 'jetpack_api_include_comment_types_count' ); // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+
+		/**
+		* Include certain comment types in comment counts in the REST API.
+		*
+		* @since $$next-version$$
+		* @module json-api
+		*
+		* @param array Array of comment types to include (default: 'comment', 'pingback', 'trackback')
+		*/
+		$include = apply_filters(
+			'jetpack_api_include_comment_types_count',
+			array( 'comment', 'pingback', 'trackback' )
```

Comment:
I think we may want to include an empty string in that default list. Indeed, up until WordPress 5.4, comments were stored with a `comment_type` of `''`:
https://core.trac.wordpress.org/ticket/49236

I think we'd want to include such comments in the default query as well.

```suggestion
		/**
		* Include certain comment types in comment counts in the REST API.
		* Note: the default array of comment types includes an empty string,
		* to support comments posted before WP 5.5, that used an empty string as comment type.
		*
		* @since $$next-version$$
		* @module json-api
		*
		* @param array Array of comment types to include (default: 'comment', 'pingback', 'trackback')
		*/
		$include = apply_filters(
			'jetpack_api_include_comment_types_count',
			array( 'comment', 'pingback', 'trackback', '' )
```